contrib/znc/template.py

```diff
@@ -0,0 +1,37 @@
+pkgname = "znc"
+pkgver = "1.9.1"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DWANT_PERL=YES",
+    "-DWANT_PYTHON=YES",
+    "-DWANT_TCL=YES",
+]
+make_cmd = "gmake"
+hostmakedepends = [
+    "cmake",
+    "gmake",
+    "perl",
+    "pkgconf",
+    "python",
+]
+makedepends = [
+    "argon2-devel",
```

Comment:
also needs boost-devel here and gettext in hostmake

Reply:
i didnt try installing with localization because afaik musl didnt have any locales per se? i thought of the boost-locale something like that package, but that was compiling some other stuff during the compile of  znc itself so i just scrapped the idea

Reply:
it does, this is application locales

Reply:
the reason it 'builds boost' is because your mr is hundreds of commits behind, and in that state the main/boost is older than the repo so it starts building the old one